sources/packages/backend/apps/api/src/services/student/student.service.ts

```diff
@@ -319,6 +332,70 @@ export class StudentService extends RecordDataModelService<Student> {
     return null;
   }
 
+  /**
+   * Check for a partial match for a student in SFAS.
+   * @param user user to be matched against.
+   * @param student student to be matched against.
+   * @param studentSIN sin to be matched against.
+   * @param auditUserId audit user for the notification.
+   * @param externalEntityManager entityManager to be used to perform the query.
```

Comment:
The entity manager will also be used to persist the notification, not only to perform a query.
The comment can be generic or more precise but right now it is not accurate.

Reply:
updated